common/src/memory_layout.rs

```diff
@@ -30,11 +30,12 @@ pub const MAN2_ORG: u32 = 0x50001800;
 pub const FHT_ORG: u32 = 0x50003000;
 pub const LDEVID_TBS_ORG: u32 = 0x50003800;
 pub const FMCALIAS_TBS_ORG: u32 = 0x50003C00;
-pub const PCR_LOG_ORG: u32 = 0x50004000;
-pub const FUSE_LOG_ORG: u32 = 0x50004400;
-pub const BOOT_STATUS_ORG: u32 = 0x500047FC;
-pub const DATA_ORG: u32 = 0x50004800;
-pub const STACK_ORG: u32 = 0x5001C000;
+pub const RTALIAS_TBS_ORG: u32 = 0x50004000;
+pub const PCR_LOG_ORG: u32 = 0x50004400;
+pub const FUSE_LOG_ORG: u32 = 0x50004800;
+pub const BOOT_STATUS_ORG: u32 = 0x50004BFC;
+pub const DATA_ORG: u32 = 0x50004C00;
```

Comment:
There's a reference to this in fmc/test-fw/test-rt/memory.x

Reply:
Also in rom/dev/tools/test-fmc/src/fmc.ld and rom/dev/tools/test-rt/src/rt.ld

Reply:
Done.